projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-row-header-group.component.ts

```diff
@@ -0,0 +1,138 @@
+    selector: 'igx-pivot-row-header-group',
+    templateUrl: './pivot-row-dimension-header-group.component.html',
+    standalone: true,
+    imports: [IgxIconComponent, NgIf, IgxPivotRowDimensionHeaderComponent, NgClass, NgStyle, IgxColumnMovingDragDirective, IgxColumnMovingDropDirective, IgxGridFilteringCellComponent, IgxPivotResizeHandleDirective, IgxHeaderGroupStylePipe]
+})
+export class IgxPivotRowHeaderGroupComponent extends IgxGridHeaderGroupComponent implements PivotRowHeaderGroupType {
+
+    /**
+     * @hidden
+     */
+    @HostBinding('style.user-select')
+    public userSelect = 'none';
+
+    constructor(private cdRef: ChangeDetectorRef,
+        @Inject(IGX_GRID_BASE) public override grid: PivotGridType,
+        private elementRef: ElementRef<HTMLElement>,
+        public override colResizingService: IgxPivotColumnResizingService,
+        filteringService: IgxFilteringService,
+        platform: PlatformUtil,
+        protected zone: NgZone) {
+        super(cdRef, grid, elementRef, colResizingService, filteringService, platform);
+    }
+
+    /**
+     * @hidden
+     * @internal
+     */
+    @Input()
+    public rowIndex: number;
+
+    @Input()
+    public set dimWidth(value: number) {
+        this.column.width = value + 'px';
+    }
+    public get dimWidth() {
+        return parseFloat(this.column.width);
+    }
+
+    public get parent() {
+        return this;
+    };
+
+    @Input()
+    public rootDimension: IPivotDimension;
+
+    @ViewChild(IgxPivotRowDimensionHeaderComponent)
+    public override header: IgxPivotRowDimensionHeaderComponent;
+
+    @HostBinding('attr.id')
+    public override get headerID() {
+        return `${this.grid.id}_-2_${this.rootDimension.memberName}_${this.visibleIndex}`;
+    }
+
+    @HostBinding('attr.title')
+    public override get title() {
+        return this.rootDimension.displayName;
+    }
+
+    /**
+     * @hidden
+     * @internal
+     */
+    public get visibleIndex(): number {
+        const rows = this.grid.rowDimensions;
+        return rows.indexOf(this.rootDimension);
+    }
+
+    @HostBinding('class.igx-grid-th--active')
+    public override get active() {
+        return false;
+    }
+
+    @HostBinding('class.asc')
+    public get sortAscendingStyle() {
+        return this.rootDimension.sortDirection === SortingDirection.Asc;
+    }
+
+    @HostBinding('class.desc')
+    public get sortDescendingStyle() {
+        return this.rootDimension.sortDirection === SortingDirection.Desc;
+    }
+
+    @HostBinding('class.igx-grid-th--sortable')
+    public get sortableStyle() {
+        return true;
+    }
+
+    @HostBinding('class.igx-grid-th--sorted')
+    public get sortedStyle() {
+        return this.rootDimension.sortDirection !== undefined && this.rootDimension.sortDirection !== SortingDirection.None;
+    }
+
+    protected override get activeNode() {
+        return null;
+    }
+
+    public override activate() { }
```

Comment:
Shouldn't the headers also become active when clicked or when navigated to via keyboard?

Reply:
I actually think I won't be that bad if the row header can become active and be part of the keyboard navigation for the column/row headers. It can also have some kb interaction like sorting with ctrl + arrow up/down like for the normal columns.

I know that's probably not a priority so it could be logged separately I guess.

Reply:
Yep, it is logged here #14249